### PR TITLE
How to login from multiple devices

### PR DESCRIPTION
**Pull request by emolinaro**

### CODE REVIEW

File: src/intro/ssh-login.rst [hunk 1]
```diff
@@ -199,10 +199,58 @@ To configure the connection, complete the following steps.
...
+
+2. Then, create the file ``~/.ssh/authorized_keys`` in your home folder, which specifies the SSH keys that can be used for logging into your user account:
+
+   .. code-block:: console
+
+      [testuser@fe-ac-02 ~]$ mkdir .ssh
+
+   .. code-block:: console
+
+      [testuser@fe-ac-02 ~]$ chmod 700 .ssh
```
Comment (mrlhansen) [inline src/intro/ssh-login.rst]: Just to make it a bit simpler, these two commands can be collapsed into one command: `install -d -m 700 ~/.ssh`

File: src/intro/ssh-login.rst [hunk 2]
```diff
@@ -199,10 +199,58 @@ To configure the connection, complete the following steps.
...
+
+   .. code-block:: console
+
+      [testuser@fe-ac-02 ~]$ touch .ssh/authorized_keys
+
+   .. code-block:: console
+
+      [testuser@fe-ac-02 ~]$ chmod 600 .ssh/authorized_keys
```
Comment (mrlhansen) [inline src/intro/ssh-login.rst]: And the same for these two: `install -D -m 600 /dev/null ~/.ssh/authorized_keys`

File: src/intro/ssh-login.rst (hunk 1 above)
Comment (emolinaro) [reply src/intro/ssh-login.rst]: ok

File: src/intro/ssh-login.rst (hunk 2 above)
Comment (emolinaro) [reply src/intro/ssh-login.rst]: ok

**Pull request merged**